zenodo/modules/pages/templates/zenodo_pages/contact_form.html

```diff
@@ -93,11 +218,96 @@
     var dropdown = document.getElementById('issue_category');
     var current_value = dropdown.options[dropdown.selectedIndex].value;
 
-    for(each in data){
-      if(data[each][0]==current_value){
-        document.getElementById('issue_text').innerHTML = data[each][1];
+    for(each in data)
+      if(each == current_value)
+        document.getElementById('issue_text').innerHTML = data[each];
+  }
+</script>
+
+<script type="text/javascript">
+  display_size = function (bytes) {
+    if (bytes == 0) {return '0.00 B';}
+    var e = Math.floor(Math.log(bytes) / Math.log(1000));
+    return (bytes / Math.pow(1000, e)).toFixed(2) + ' ' + ' KMGTP'.charAt(e) + 'B';
+  };
+
+  // Display uploaded files and it's size.
+  // Throws error if size is greater than max attachment size.
+  (function(window) {
+    function triggerCallback(e, callback) {
+      if(!callback || typeof callback !== 'function') {
+        return;
+      }
+      var files;
+      if(e.dataTransfer) {
+        files = e.dataTransfer.files;
+      } else if(e.target) {
+        files = e.target.files;
       }
+      callback.call(null, files);
     }
-  }
+
+    function makeDroppable(ele, ele1, callback) {
+      var input = document.getElementById('attachments');
+      input.style.display = 'none';
+
+      input.addEventListener('change', function(e) {
+        triggerCallback(e, callback);
+      });
+
+      ele.addEventListener('dragover', function(e) {
+        e.preventDefault();
+        e.stopPropagation();
+      });
+
+      ele.addEventListener('dragleave', function(e) {
+        e.preventDefault();
+        e.stopPropagation();
+      });
+
+      ele.addEventListener('drop', function(e) {
+        e.preventDefault();
+        e.stopPropagation();
+        triggerCallback(e, callback);
+      });
+
+      ele1.addEventListener('click', function() {
+        input.value = null;
+        input.click();
+      });
+    }
+    window.makeDroppable = makeDroppable;
+  })(this);
+  (function(window) {
+    makeDroppable(window.document.querySelector('html'),
+                  window.document.querySelector('#upload'),
+                  function(files) {
+      var numFiles = files.length;
+      var string = '';
+      if(numFiles == 0)
+        return ;
+      string = '<strong>' + numFiles + ' files selected.</strong>';
+      var totalSize = 0;
+      for(var i=0;i<numFiles;i++){
+        totalSize += files[i].size;
+        string += '<br /><small>' + files[i].name + ' &nbsp&nbsp ' +
+        display_size(files[i].size) + '</small>';
+      }
+
+      document.getElementById('upload_info').innerHTML = string;
+
+      if(totalSize > {{ max_file_size }}){
+        $('.field-attachments').addClass('has-error');
+        document.getElementById('error-attachments').innerHTML =
+        'File size exceeded. Please add URL to file instead.';
```

Comment:
Better text: `File size exceeded. Please add URLs to the files or make a smaller selection.`